### PR TITLE
small readability to codegen logging

### PR DESCRIPTION
**Pull request by tippenein**
also nab the last instances of the "independent" typo

### CODE REVIEW

File: app/Main.hs [hunk 1]
```diff
@@ -111,22 +112,22 @@ run' opts =
...
-                                  <> li_name Elm.libraryInfo <> " version " <> li_version Elm.libraryInfo
-                             mapM_ (runner dir Elm.makeModule Elm.makeFileName) readyModules
-                      _ <- T.forM (o_psOutDir opts) $ \dir ->
-                          do T.putStrLn $
-                                  "Required PureScript library is "
-                                  <> li_name PS.libraryInfo <> " version " <> li_version PS.libraryInfo
-                             mapM_ (runner dir PS.makeModule PS.makeFileName) readyModules
-                      return ()
+                   do
+                     forM_ (o_hsOutDir opts) $ \dir -> do
```
Comment (agrafix) [inline app/Main.hs]: I would like to keep the coding style consistent, so the `do` should go to the next line

File: app/Main.hs [hunk 2]
```diff
@@ -111,22 +112,22 @@ run' opts =
...
+                         printLibraryInfo Elm.libraryInfo
+                         mapM_ (runner dir Elm.makeModule Elm.makeFileName) readyModules
+                     forM_ (o_psOutDir opts) $ \dir -> do
+                         printLibraryInfo PS.libraryInfo
+                         mapM_ (runner dir PS.makeModule PS.makeFileName) readyModules
+                     return ()
+
+printLibraryInfo :: LibraryInfo -> IO ()
+printLibraryInfo li = T.putStrLn $
+    "Required " <> li_type li <>
```
Comment (agrafix) [inline app/Main.hs]: We should probably add a space after the library type

File: app/Main.hs [hunk 3]
```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE TemplateHaskell #-}
 module Main where
 
+import Control.Monad (forM_)
```
Comment (agrafix) [inline app/Main.hs]: I usually try to group imports into two groups:
- imports coming from this packages
- all other imports

**Comment by agrafix** [comment]
Thanks for the PR! Could you please fix the warning? I always build using `stack build --fast --pedantic`

**Comment by tippenein** [comment]
Certainly. I didn't realize it did the tests in multiple chunks, so I only saw the passing Elm test.
I'll fix the spec

**Comment by tippenein** [comment]
Some of the style edits were done by [stylish-haskell](https://github.com/jaspervdj/stylish-haskell) with these settings: http://lpaste.net/154635

**Pull request merged**